<!--
  2 failure:  :disappointed_rel..., :microscope: Ther...
  0 warning: 
  2 messages
  
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  :microscope: There are library changes, but no changes to the unit tests. That's OK as long as you're refactoring existing code, but will require an admin to merge this PR. Please see <a href="https://github.com/miniman42/titanium_mobile/blob/TIMOB26399_7_4_X_Fix_Backport/README.md#unit-tests">README.md#unit-tests</a> for docs on unit testing.
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@7.4.1 test /Users/build/jenkins/workspace/dk_titanium_mobile_PR-10355-2O7WANYKHKBIJPNEPEDWN4N4XUO274NC7MAWYCV5TTHLMSVH6JPA
> grunt

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

Running "checkFormat:ios" (checkFormat) task
Fatal error: Error: Formatting incorrect on "iphone/Classes/TiApp.m", proposed changes: <?xml version='1.0'?>
<replacements xml:space='preserve' incomplete_format='false'>
<replacement offset='22277' length='0'> </replacement>
<replacement offset='22278' length='9'>&#10;      </replacement>
<replacement offset='22416' length='9'>&#10;      </replacement>
<replacement offset='22458' length='9'>&#10;      </replacement>
</replacements>

npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :tada: Another contribution from our awesome community member, miniman42! Thanks again for helping us make Titanium SDK better. :thumbsup:
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/iphone/Classes/TiApp.m
+++ b/iphone/Classes/TiApp.m
@@ -623,10 +623,10 @@ TI_INLINE void waitForMemoryPanicCleared(); //WARNING: This must never be run on
       [responseInfo setValue:((UNTextInputNotificationResponse *)response).userText forKey:UIUserNotificationActionResponseTypedTextKey];
     }
     if ([UNNotificationDefaultActionIdentifier isEqualToString:response.actionIdentifier]) {
-        [self application:[UIApplication sharedApplication] didReceiveRemoteNotification:response.notification.request.content.userInfo];
-        completionHandler();
+      [self application:[UIApplication sharedApplication] didReceiveRemoteNotification:response.notification.request.content.userInfo];
+      completionHandler();
     } else {
-        [self application:[UIApplication sharedApplication] handleActionWithIdentifier:response.actionIdentifier forRemoteNotification:response.notification.request.content.userInfo withResponseInfo:responseInfo completionHandler:completionHandler];
+      [self application:[UIApplication sharedApplication] handleActionWithIdentifier:response.actionIdentifier forRemoteNotification:response.notification.request.content.userInfo withResponseInfo:responseInfo completionHandler:completionHandler];
     }
   } else {
     //NOTE Local notifications should be handled similar to BG above which ultimately calls handleRemoteNotificationWithIdentifier as this will allow BG Actions to execute.

--- a/iphone/Classes/TiApp.m
+++ b/iphone/Classes/TiApp.m
@@ -622,7 +622,7 @@ TI_INLINE void waitForMemoryPanicCleared(); //WARNING: This must never be run on
       responseInfo = [NSMutableDictionary dictionary];
       [responseInfo setValue:((UNTextInputNotificationResponse *)response).userText forKey:UIUserNotificationActionResponseTypedTextKey];
     }
-    if ([UNNotificationDefaultActionIdentifier isEqualToString:response.actionIdentifier]){
+    if ([UNNotificationDefaultActionIdentifier isEqualToString:response.actionIdentifier]) {
         [self application:[UIApplication sharedApplication] didReceiveRemoteNotification:response.notification.request.content.userInfo];
         completionHandler();
     } else {

